pkg/liqo-controller-manager/namespaceMap-controller/ensure_remote_namespaces.go

```diff
@@ -177,3 +188,20 @@ func (r *NamespaceMapReconciler) namespaceMapDeletionProcess(ctx context.Context
 	}
 	return fmt.Errorf("remote namespaces deletion phase in progress")
 }
+
+// checkRemoteNamespacePrivileges checks the number of roleBindings inside the remote namespace to understand if the
+// virtual kubelet will have the right privileges on that namespace.
+func checkRemoteNamespacePrivileges(ctx context.Context, cl client.Client, remoteNamespaceName string) error {
+	roleBindingList := &rbacv1.RoleBindingList{}
+	if err := cl.List(ctx, roleBindingList, client.InNamespace(remoteNamespaceName)); err != nil {
+		klog.Errorf("%s -> unable to list roleBindings in the remote namespace '%s'", err, remoteNamespaceName)
+		return err
+	}
+	if len(roleBindingList.Items) < 2 {
+		err := fmt.Errorf("virtual kubelet will not have the necessary privileges on the remote namespace '%s'",
+			remoteNamespaceName)
+		klog.Error(err)
+		return err
+	}
+	return nil
```

Comment:
I would personally try to get the rolebinding by name or label (depending on how it is configured) and not rely on the number, since it may lead to false positives.

Reply:
Yes, I can use a label to List the roleBindings, do you agree @aleoli ?